### ISSUE TITLE
Doesn't work inside tmux

### ISSUE THREAD

**Issue by wookayin** [ojroques/nvim-osc52#17]
This plugin works well outside tmux, but not inside a tmux session. I've set up tmux-osc52 well (https://github.com/tmux/tmux/wiki/Clipboard#quick-summary) and I can confirm it works:

```
# (inside a tmux)
esc="\033]52;c;$(base64 <<< 'hello world')\a"
esc="\033Ptmux;\033$esc\033\\" # tmux needs sequence escapement
printf "$esc"
```

The OSC52 sequence need to be wrapped and escaped properly to enable passthrough. However, the plugin doesn't seem to be aware of tmux escape sequences:

https://github.com/ojroques/nvim-osc52/blob/main/lua/osc52.lua#L58

**Comment by ojroques** [comment]
Hello,
You should follow this guide first for tmux: https://github.com/tmux/tmux/wiki/Clipboard#quick-summary
tl;dr: put `set -s set-clipboard on` (not external) in your tmux config.

**Comment by wookayin** [comment]
I see, that works. The guide looked like to suggest to use `external`. It's mentioned in your README; thanks.

**Comment by twmht** [comment]
@wookayin 

can you share your .tmux.conf because i also have issues with tmux.
